resources/logical_interconnect_group.rb

```diff
@@ -38,15 +40,57 @@ def load_lig(item)
         raise "#{up[:networkType]} #{network_name} not found" unless net.retrieve!
         up.add_network(net)
       end
-      uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
+      parsed_uplink_info[:connections].each { |link| up.add_uplink(link[:bay], link[:port]) }
       item.add_uplink_set(up)
     end
     item
   end
+
+  # like? function specific to the interconnectMapTemplate resource
+  def interconnect_map_like?(item, another_item)
```

Comment:
Yep, that's my point, we could not add anything else to the resource.rb `like?` because we don't know if these things are general, and because for each resource there may be some particularities, so overriding the resources would be the way to handle this.